src/util/ConcurrentCache.h

```diff
@@ -157,10 +157,14 @@ class ConcurrentCache {
   using Value = typename Cache::value_type;
   using Key = typename Cache::key_type;
 
-  ConcurrentCache() requires std::default_initializable<Cache> = default;
+  // TODO<joka921>: Default constructor, needs some proper solution
+  ConcurrentCache() QL_CONCEPT_OR_NOTHING(
+      requires std::default_initializable<Cache>) = default;
   /// Constructor: all arguments are forwarded to the underlying cache type.
-  template <typename CacheArg, typename... CacheArgs>
-  requires(!std::same_as<ConcurrentCache, std::remove_cvref_t<CacheArg>>)
+  // TODO<joka921>: Temporary solution until proper macros are made
+  template <typename CacheArg, typename... CacheArgs,
+            typename = std::enable_if_t<
+                !std::same_as<ConcurrentCache, std::remove_cvref_t<CacheArg>>>>
```

Comment:
There is a macro for constructors, I'll have to look at it, but maybe also CPP_template works for such function,s haven't tried it yet.